docs/man-openmpi/man3/MPI_Waitany.3.rst

```diff
@@ -88,20 +88,20 @@ special value for the *status* argument.
 
        CALL MPI_COMM_SIZE(comm, size, ierr)
        CALL MPI_COMM_RANK(comm, rank, ierr)
-       IF(rank .GT 0) THEN         ! client code
-           DO WHILE(.TRUE.)
+       IF(rank > 0) THEN         ! client code
+           DO
```

Comment:
If we remove the "while true", is there a reason for the DO block at all?

Reply:
Yes! :)
```fortran
Do

End do
! is equivalent to
do while (.true.)

end do
```
so no need to complicate things for the reader.

Reply:
As a non-Fortran programmer, that's a pretty subtle point if you don't know that those 2 are equivalent. 😄